### ISSUE TITLE
Quick Picks dont have Related albums & Similar artist

### ISSUE THREAD

**Issue by faustaahmad** [fast4x/RiMusic#504]
### Steps to reproduce the bug

Enable All Quick Picks
Most played song or Last ~~played~~ added song must not registered in Youtube Music, mine from Youtube not music
Settings > Quick Picks > Tips:By Most played song

### Expected behavior

Related albums & Similar artist should be visible

### Actual behavior

I doesnt appear, perhaps because of the song isnt registered in Youtube Music

### Screenshots/Screen recordings

![](https://github.com/fast4x/RiMusic/assets/45221211/9c46b75f-d4a3-4357-98e0-5ecb9bfd87e6)

### RiMusic version

0.6.19

### Android version

Android 11

### Suggestion

Fallback to Most played song that are registered in Youtube Music

Since radio works, and mixed with some of songs that are registered on Youtube Music
Try to find closest music on radio that are registered on Youtube Music
Then use that song as source for Tips, Related albums, and Similar artist
While still displaying that one Most played song or Last played song

**Comment by fast4x** [comment]
Do you have enabled It in Quick Picks settings?

**Comment by faustaahmad** [comment]
Obviously all of 'em

**Comment by NEVARLeVrai** [comment]
> Do you have enabled It in Quick Picks settings?

I have the same issue with my godfather actually listens to youtube videos so no album or artist is offered I think it's not a issue

**Comment by faustaahmad** [comment]
No sound, all Quick Picks related settings enabled

[456d-8cff-41cc-8104-0855109e15f5.webm](https://github.com/fast4x/RiMusic/assets/45221211/46139877-7ff5-46da-8bf8-8ee8fb6f5b1b)

**Comment by fast4x** [comment]
Check in settings > Quick Picks tab

**Comment by NEVARLeVrai** [comment]
If u listen from YT video you don't have related albums ? I think it's it's YT music you have

**Comment by faustaahmad** [comment]
I dont think it's neccesary for me to give this information, as it's far too simple & enabled by default

![Screenshot_20240128-210614](https://github.com/fast4x/RiMusic/assets/45221211/48b4d12c-1918-4803-ac65-b8287dfb93e2)

Try music on Youtube, but not registered on Youtube music
Well... even if it registered, still wont work anyway IF using Last played

As far my test goes, it's not Last played but Last added songs to the library that's played

**Comment by fast4x** [comment]
Try to refresh this settings try again

**Comment by faustaahmad** [comment]
I tried enable/disable Quick Picks settings still the same
However, if I reset Quick Picks the problem will temporarily fixed the problem for Last played
But it'll reset Most played song too, which sad

Also, I mentioned in #505 That Last played is bugged (see my previous comment in this issue)

**Comment by fast4x** [comment]
Ok thanks, I'll investigate it

**Comment by fast4x** [comment]
Improved most and last played
Available from version 0.6.20

**Comment by faustaahmad** [comment]
Tested 0.6.20
Last Played behavior still works like before (both in Quick Picks & ⬈ Last played)
They use Last added song that is played, instead of Last played song regardless of new/old (idk how to explain)

Since I cant explain well
Example Quick Picks:
```diff
Assume
Last Played on Quick Picks (QP)
Min listening time = 0s
A, B, C as New songs

Play A ~> QP = A
Play B ~> QP = B
- Play A ~> QP = B (Current behavior)
+ Play A ~> QP = A (Desired behavior)
Play C ~> QP = C
- Play A ~> QP = C (Current behavior)
+ Play A ~> QP = A (Desired behavior)
```

As for ⬈ songs (Last played), it works like Quick Picks as I explained above
```
[C
,B
,A
,Unchanged List from Last added songs]
```
But I'm sure if one of 'em get fixed, the other one will be fixed too
